### DISCUSSION TITLE
Graphite compability status

### DISCUSSION THREAD

**Discussion by Razikus** [grafana/mimir#6027] [General]
Recently I saw that Metrictank is deprecated. 
How can I achieve graphite compability with grafana Mimir?

**Comment by pstibrany** [comment]
Grafana Mimir does not support Graphite natively.

To ingest Graphite data into Mimir, you can use Graphite Proxy writes: https://github.com/grafana/mimir-proxies/blob/main/cmd/graphite-proxy-writes/README.md

Grafana Labs also offers Grafana Enterprise Metrics product built on Grafana Mimir that offers full Graphite support: https://grafana.com/docs/enterprise-metrics/latest/graphite/

If Metrictank works for you, you can also keep using it.

**Comment by Razikus** [comment]
how can i read graphite from grafana mimir? 

In mimir there is following comment:

```
As of August 2023, Grafana is no longer maintaining this repository. Our primary compatibility with Graphite is provided by [carbonapi](https://github.com/go-graphite/carbonapi), using [Mimir](https://github.com/grafana/mimir) as our backing database.
```

**Comment by npazosmendez** [comment]
Hello @Razikus! If using the mimir-proxy for graphite, you can query the metrics using PromQL directly, following the translation described in [the docs](https://github.com/grafana/mimir-proxies/blob/main/cmd/graphite-proxy-writes/README.md).
The ability to query those metrics (the ones ingested with the mimir-proxy) using Graphite language is provided by Grafana Enterprise Metrics, as Peter mentioned.

**Comment by Razikus** [comment]
thanks, so i can't really just switch to mimir - it requires very big migration :/ great
